pkgs/development/python-modules/nanobind/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib, buildPythonPackage, pythonOlder, fetchFromGitHub, cmake, eigen, ninja
+, scikit-build, pytestCheckHook, numpy, scipy, torch, jax, jaxlib, tensorflow
+, setuptools }:
+buildPythonPackage rec {
+  pname = "nanobind";
+  version = "1.8.0";
```

Comment:
Version [1.9.0](https://github.com/wjakob/nanobind/releases/tag/v1.9.0) is out and the derivation seems to work if only the hash is changed.